src/Rings/groebner.jl

```diff
@@ -110,48 +138,48 @@ julia> H = groebner_basis(I, ordering=lex(gens(R)))
  6*x^2 - y^3
 ```
 """
-function groebner_basis(I::MPolyIdeal; ordering::MonomialOrdering = default_ordering(base_ring(I)), complete_reduction::Bool=false, enforce_global_ordering::Bool = true)
-    groebner_assure(I, ordering, complete_reduction, enforce_global_ordering)
```

Comment:
Why don't we need `groebner_assure` here anymore?

Reply:
The logic of groebner_assure is now in std_basis().